src/widgets/connector.ts

```diff
@@ -112,7 +124,7 @@ export class ConnectorView extends DOMWidgetView {
                 <div class="block">
                     <div class="create-new-connection" id="createNewConnectionButton">
                         <div class="icon"></div>
-                        <div>
+                        <div id="createNewConnection">
```

Comment:
If we use `id` it may cause unpredictable behavior if the user runs `ConnectorWidget()` in more than one cell in a single notebook since now we'll have multiple DOM elements with the same id. 
  suggestion: we can add a unique id for the instance and use it as a suffix for the id, i.e.: `id=createNewConnection_uniqueWidgetId` or we can throw an error if the widget already exists in the notebook.

Reply:
why would this cause issues? our widget code doesn't rely on selecting (or modifying) this element by ID. I added one because it allows me to test quickly (to simulate a click on this element).

In any case, I ran a quick test: the existing implementation has something like this in `handleCreateNewConnectionClick` (it's selecting by ID)

```ts
// hide connectionsContainer
(<HTMLElement>this.el.querySelector("#connectionsContainer")).style.display = "none";

// show newConnectionContainer
(<HTMLElement>this.el.querySelector("#newConnectionContainer")).style.display = "block";
```

I created a notebook with two connector widgets, I thought that when clicking on "new connection", this would cause both widgets to change the view (since this moves from the list of connections to the new connection form), but it didn't, only the widget I clicked on was modified (unsure how this is working under the hood, as I also expect this to cause issues)

![2023-08-27 8 53 03 p m](https://github.com/ploomber/jupysql-plugin/assets/989250/86c177f9-edbb-44ff-ab6d-513fcc3f6778)

Reply:
Yes, I also tried to play with it but it seems to work without any errors. So let's keep it as is then.